### ISSUE TITLE
[docs][workflows]: add more info & controls for Working with GIFs

### ISSUE THREAD

**Issue by marcysutton** [gatsbyjs/gatsby#14755]
## Summary

Continuing on from https://github.com/gatsbyjs/gatsby/issues/14529, the new page on [Working with GIFs](https://www.gatsbyjs.org/docs/working-with-gifs/) has two improvements to be made:

1. Add controls to pause the animating GIF on page load to not pose accessibility problems for users with motion sensitivity, perhaps with [react-gif-player](https://www.npmjs.com/package/react-gif-player);
2. Explain the phrase "beware that flashing and autoplaying GIFs can cause accessibility problems" with a bit more detail. We could certainly mention it violates WCAG 2.2.2 for "pause, stop, hide". Some resources for reference material: 
    - https://source.opennews.org/articles/motion-sick/
    - https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html

We need to do this as part of our own commitment to accessibility in Gatsby, but we also want to take every opportunity to teach best practices–and this is a best practice for working with animation and motion on the web.
